src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -174,6 +167,7 @@ class HumanIK
     object */
 
     int m_nrDoFs; /** Number of Joint Degrees of Freedom */
+    bool tPose{false}; /** Flag for resetting the integrator state */
```

Comment:
```suggestion
    bool m_tPose{false}; /** Flag for resetting the integrator state */
```

I would rename this variable to be consistent with the rest of the code by adding `m_` in front.

Reply:
Done with commit https://github.com/ami-iit/biomechanical-analysis-framework/pull/31/commits/575da1ffecd6145855cad2d5019c3e90325b45b3